src/table/table.scss

```diff
@@ -332,6 +332,23 @@
   &.iui-negative {
     @include iui-table-cell-status($status: negative);
   }
+
+  &[contenteditable] {
+    outline-offset: -1px;
```

Comment:
Without this, outline goes outside cell. Vertically it is ok as it merges with the borders but horizontally it goes outside the bound.